requirements.txt

```diff
@@ -7,3 +7,15 @@ git+https://github.com/SymbiFlow/symbiflow-xc-fasm.git
 git+https://github.com/capnproto/pycapnp.git
 git+https://github.com/SymbiFlow/symbiflow-xc-fasm2bels.git
 third_party/python-fpga-interchange
+
+# Litex
+third_party/litex
+third_party/litex-boards
+third_party/litedram
+third_party/liteeth
+third_party/liteiclink
+third_party/pythondata-cpu-vexriscv
+third_party/pythondata-software-compiler_rt
+third_party/migen
```

Comment:
It may be worth adding `-e` before each of them. This will make PIP install them as "editable" and allow easier debugging in the future. Same may go with python-fpga-interchange

Reply:
Right, I'll open a follow-up PR, so not to block this one